### ISSUE TITLE
launchYoutubeApp locks after launching the app on chromecast

### ISSUE THREAD

**Issue by shaikhmohsink** [ConnectSDK/Connect-SDK-Android-Core#82]
Hi,

I am trying to integrate the sample app using the updated connect-sdk library,
below is the code
getLauncher().launchYouTube("bB3aZzsZSOE", (float)0.0,  new Launcher.AppLaunchListener() {

                    @Override
                    public void onSuccess(LaunchSession session) {
                        testResponse =  new TestResponseObject(true, TestResponseObject.SuccessCode, TestResponseObject.Launched_Youtube);
                        Toast.makeText(MainActivity.this, "ChromeCastResponse:-"+testResponse.httpResponseCode+", "+testResponse.responseMessage+", "+testResponse.isSuccess, Toast.LENGTH_LONG).show();
                        setRunningAppInfo(session);
                    }

                    @Override
                    public void onError(ServiceCommandError error) {
                        Toast.makeText(MainActivity.this, "Error casting to chromecast"+error.getMessage(), Toast.LENGTH_LONG).show();
                    }
                });

this opens the youtube app and after that nothing happens, no video is played at all, where as in the respose, i get the http status as 200 and isSuccess as true.

**Comment by iadnan** [comment]
@shaikhmohsink Which TV or media device are trying to open the app?

**Comment by shaikhmohsink** [comment]
I am using a samsung monitor, i even tried the same on a samsung TV
And i had bought the chromecast device last december

**Comment by shaikhmohsink** [comment]
Hi iadnan,
Just wanted to check if you are able to use the latest version of connect sdk for android and implement the launchYoutubeApp method, and see anything playing on any of the media device like TV or monitor or anything at all

eagerly waiting for reply

**Comment by iadnan** [comment]
@shaikhmohsink I believe you are trying to open the app on Chromecast device?. We tried to reproduce the issue with you tube app. When trying to launch a you tube app , the app doesnt do anything. We will look into it. Thanks.

**Comment by shaikhmohsink** [comment]
Yes I am trying to open the app on chromecast device, and apparently it is not working on both iOS and Android. This is just in case of youtube app. rest of the videos are playing fine on the chromecast device.

Thanks for your help :)

**Comment by shaikhmohsink** [comment]
Hi,

Any updates on this issue?

**Comment by Gioggiolo** [comment]
Hi, Same issue here. Has anything changed in youtube app protocol?

**Comment by shaikhmohsink** [comment]
Hey,

I just saw this new message on my browser saying 
"dmayboroda was unassigned by shaikhmohsink on 11 Jan"

Not really sure how did this happened, I just logged in to check if there was any solution found for this problem. Can anyone answer this please?

**Comment by bhargavakumark** [comment]
I looked at the node package https://www.npmjs.com/package/peer-dial which does this. It launches the youtube app using

                    dialDevice.launchApp("YouTube","v=YE7VzlLtp-4", "text/plain", function (launchRes, err)

Without using any pairing key. I removed the pairingkey code in DIALService.java, and YouTube can now be launched

**Comment by leo81267** [comment]
Hi , 

I'm trying Connect SDK with Xbox. The solution suggested above (removing the pairing code) works successfully for all the other devices - Chromecast , Nexus player , Android TV etc But not with the Xbox ?

I tried to send the pairing code too , but it launchs the app and goes to home screen of youtube.

Any idea or thoughts on this ?

**Comment by baitsaikiransudhir** [comment]
Hello, any update on this. I'm using this and it just opens the homescreen of youtube. It does not open the video.

**Comment by waqarkhan1137** [comment]
i am using Connect SDK but when I try to connect with my Android smart TV its directly connect without pairing .. and I want to pair with my Smart Tv using connect SDK
